cairo/tests/utils/serde.py

```diff
@@ -371,6 +378,29 @@ def serialize_tuple(self, ptr, item_scope):
         raw = self.serialize_pointers(f"Tuple{item_scope}Struct", tuple_struct_ptr)
         return tuple(self.serialize_list(raw["value"], item_scope, list_len=raw["len"]))
 
+    def serialize_sequence(self, ptr, item_scope):
+        sequence_struct_ptr = self.serialize_pointers(f"Sequence{item_scope}", ptr)[
+            "value"
+        ]
+        raw = self.serialize_pointers(
+            f"Sequence{item_scope}Struct", sequence_struct_ptr
+        )
+        return list(self.serialize_list(raw["value"], item_scope, list_len=raw["len"]))
+
+    def serialize_enum(self, ptr, item_scope):
+        enum_struct_ptr = self.serialize_pointers(f"Enum{item_scope}", ptr)["value"]
```

Comment:
so this is now the generic way of modelling an Enum in cairo ?

Reply:
the "generic" way I've come up with, which comes with this kind of definition

```
struct EnumMyEnum {
    value: EnumMyEnumStruct*,
}

struct EnumMyEnumStruct {
    sequence: Sequence,
    bytes: Bytes,
    // other options
}
```